testdata/baselines/reference/submodule/compiler/noUnusedLocals_typeParameterMergedWithParameter.errors.txt.diff

```diff
@@ -24,8 +22,9 @@
 -!!! error TS6133: 'T' is declared but its value is never read.
      
      function useParam<T>(T: number) {
-                      ~~~
+-                     ~~~
 -!!! error TS6133: 'T' is declared but its value is never read.
++                      ~
```

Comment:
The new range here appears to be unexpected, though I can't say I like where it used to be (including the angle brackets) either.

Reply:
Yes, that's intended.

Reply:
Also, the old compiler talks about "reading" type parameters, which is nonsense for non-values.